Mathlib/MeasureTheory/MeasurableSpace.lean

```diff
@@ -275,6 +275,19 @@ theorem measurable_const' {f : β → α} (hf : ∀ x y, f x = f y) : Measurable
   apply hf
 #align measurable_const' measurable_const'
 
+-- porting note: Attribute not yet supported
+-- @[measurability]
+theorem measurable_nat_cast [NatCast α] (n : ℕ) : Measurable (n : β → α) :=
```

Comment:
Should this be `natCast`? I realize we're inconsistent within mathlib4, but perhaps you have a preference.

Reply:
[Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/naming.20conventions.3A.20natCast.20vs.20nat_cast/near/346323728)